Tests/DatadogTests/Datadog/RUM/RUMFeatureTests.swift

```diff
@@ -167,67 +160,4 @@ class RUMFeatureTests: XCTestCase {
         XCTAssertTrue(tagsItem?.value?.contains("variant:\(randomVariant)") == true)
     }
-
-    // MARK: - HTTP Payload
-
-    func testItUsesExpectedPayloadFormatForUploads() throws {
-        let server = ServerMock(delivery: .success(response: .mockResponseWith(statusCode: 200)))
-        let httpClient = HTTPClient(session: server.getInterceptedURLSession())
-
-        let core = DatadogCore(
-            directory: temporaryCoreDirectory,
-            dateProvider: SystemDateProvider(),
-            initialConsent: .granted,
-            userInfoProvider: .mockAny(),
-            performance: .combining(
-                storagePerformance: StoragePerformanceMock(
-                    maxFileSize: .max,
-                    maxDirectorySize: .max,
-                    maxFileAgeForWrite: .distantFuture, // write all events to single file,
-                    minFileAgeForRead: StoragePerformanceMock.readAllFiles.minFileAgeForRead,
-                    maxFileAgeForRead: StoragePerformanceMock.readAllFiles.maxFileAgeForRead,
-                    maxObjectsInFile: 3, // write 3 spans to payload,
-                    maxObjectSize: .max
-                ),
-                uploadPerformance: UploadPerformanceMock(
-                    initialUploadDelay: 0.5, // wait enough until events are written,
-                    minUploadDelay: 1,
-                    maxUploadDelay: 1,
-                    uploadDelayChangeRate: 0
-                )
-            ),
-            httpClient: httpClient,
-            encryption: nil,
-            contextProvider: .mockAny(),
-            applicationVersion: .mockAny()
-        )
-        defer { core.flushAndTearDown() }
-
-        // Given
-        let featureConfiguration: RUMFeature.Configuration = .mockAny()
-        let feature: RUMFeature = try core.create(
-            configuration: createRUMConfiguration(configuration: featureConfiguration),
-            featureSpecificConfiguration: featureConfiguration
-        )
-        core.register(feature: feature)
-
-        let writer = feature.storage.writer(for: .granted, forceNewBatch: false)
-        writer.write(value: RUMDataModelMock(attribute: "1st event"))
-        writer.write(value: RUMDataModelMock(attribute: "2nd event"))
-        writer.write(value: RUMDataModelMock(attribute: "3rd event"))
-
-        let payload = try XCTUnwrap(server.waitAndReturnRequests(count: 1)[0].httpBody)
-
-        // Expected payload format:
-        // ```
-        // event1JSON
-        // event2JSON
-        // event3JSON
-        // ```
-
-        let eventMatchers = try RUMEventMatcher.fromNewlineSeparatedJSONObjectsData(payload)
-        XCTAssertEqual((try eventMatchers[0].model() as RUMDataModelMock).attribute, "1st event")
-        XCTAssertEqual((try eventMatchers[1].model() as RUMDataModelMock).attribute, "2nd event")
-        XCTAssertEqual((try eventMatchers[2].model() as RUMDataModelMock).attribute, "3rd event")
```

Comment:
suggesstion/ This test was asserting the payload format in RUM. Now when we remove it, we lost this coverage.

When I forge this change in `RUM/Feature/RequestsBuilder.swift`:
```diff
    /// The RUM request body format.
-    let format = DataFormat(prefix: "", suffix: "", separator: "\n")
+    let format = DataFormat(prefix: "---", suffix: "---", separator: "-")
```
all tests pass green ⚠️. It seems that need to add tests for RUM `RequestsBuilder`, no?

Reply:
Good catch! I've re-introduce this test 👍